docs/source/usage.rst

```diff
@@ -296,6 +296,10 @@ Variable interpolation
 ----------------------
 
 OmegaConf supports variable interpolation. Interpolations are evaluated lazily on access.
+Interpolations are absolute by default.
+Relative interpolation are prefixed by one or more dots:
+The first dot denotes the level of the node itself and additional dots are going up the parent hierarchy.
+e.g. ..foo goes to the foo node of the parent of the containing node.
```

Comment:
* Should this be under `Config node interpolation` below instead? (since this prefix doesn't apply to custom interpolations)
* The formatting / case could be improved to make it look better when compiled to HTML.
* "the foo node of the parent of the containing node" => a bit confusing (at least to me), would that make sense: "the `foo` sibling of the parent of the current node"?

Reply:
formatted. looks like this now:

![image](https://user-images.githubusercontent.com/376455/108139073-24bfd100-7074-11eb-82b5-8a891fa2d71a.png)